server.js

```diff
@@ -8,8 +8,13 @@ const Dispatcher = require('./lib/dispatcher');
 const PORT = process.env.PORT || 3000;
 const webhook = createHandler({path: '/', secret: process.env.WEBHOOK_SECRET || 'development'});
 
+var debug = require('debug')('http')
```

Comment:
> `http`

I think this is supposed to be `probot`. This is the namespace you use if multiple dependencies in the app use the debug package and you only want output from one of them.  e.g. `DEBUG=probot script/server` would only output for this project.

```
const debug = require('debug')('my-namespace')  
const name = 'my-app'  
debug('booting %s', name)  
```